File: packages/android/src/main/kotlin/gov/civui/components/CivMetricGroup.kt
// CivUI — CivMetricGroup for Jetpack Compose
// Responsive grid wrapper for `civ-metric-tile` children. Mirrors
// civ-metric-group.
//
// Placeholder body — the prop surface satisfies schema parity. The
// Compose implementation should use LazyVerticalGrid with
// GridCells.Adaptive. See .claude/rules/audit-debt.md →
// "Native platform implementation pass".

package gov.civui.components

import androidx.compose.foundation.layout.Column
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier

@Composable
fun CivMetricGroup(
    columns: Int = 4,
    modifier: Modifier = Modifier,
    content: @Composable () -> Unit = {},
) {
    Column(modifier = modifier) {
        // Placeholder — see audit-debt.md for native implementation status.
        content()
    }
}

File: packages/ios/Sources/CivUI/CivMetricGroup.swift
// CivUI — CivMetricGroup for SwiftUI
// Responsive grid wrapper for `civ-metric-tile` children. Mirrors
// civ-metric-group.
//
// Placeholder body — the prop surface satisfies schema parity. The
// SwiftUI implementation should use LazyVGrid with adaptive size
// classes (1 / 2 / `columns` per breakpoint). See
// .claude/rules/audit-debt.md → "Native platform implementation pass".

import SwiftUI

public struct CivMetricGroup: View {
    /// Maximum number of columns on desktop breakpoints.
    public var columns: Int

    public init(columns: Int = 4) {
        self.columns = columns
    }

    public var body: some View {
        EmptyView()
    }
}
